packages/js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@novu/js",
+  "version": "0.24.2",
+  "repository": "https://github.com/novuhq/novu",
+  "description": "Novu's JavaScript SDK for building custom inbox notification experiences",
+  "author": "",
+  "license": "ISC",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "files": [
+    "dist/*"
+  ],
+  "sideEffects": false,
+  "private": true,
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "start": "pnpm run build -- --watch --sourcemap",
+    "build": "tsup",
+    "lint": "eslint --ext .ts,.tsx src",
+    "test": "jest"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.2.3",
+    "@types/node": "^18.11.12",
+    "jest": "^29.3.1",
+    "ts-jest": "^29.0.3",
+    "tsup": "^8.0.2",
+    "typescript": "4.9.5"
+  },
+  "dependencies": {
+    "@novu/client": "workspace:*",
+    "@novu/shared": "workspace:*",
```

Comment:
This feels dangerous dependency. Is @novu/shared treeshable?

 Like I said before, starting from a clean slate and defining the new JS SDK with only the necessary, frontend fields. This exercise will also dictate the necessary Novu API changes to make it leaner.
 
 I am happy to discuss this offline on Monday.

Reply:
yes it should be... and we only use types with `import type` statement
agree, ideally we want to have a shared interface between FE and BE